style/README.md

```diff
@@ -39,9 +39,6 @@ Formatting
 * Don't include spaces after `(`, `[` or before `]`, `)`.
 * Don't misspell.
 * Don't vertically align tokens on consecutive lines.
-* If you break up an argument list, keep the arguments on their own lines and
-  closing parenthesis on its own line. [Example 1][multi-line arguments]
```

Comment:
Should we remove `multi-line arguments` and `long method signature` while we're at it? Or are they referenced elsewhere?